waku/v2/node/jsonrpc/debug_api.nim

```diff
@@ -1,3 +1,5 @@
+{.push raises: [Defect, CatchableError].}
```

Comment:
`CatchableError` means all types of exceptions are allowed, should not we limit them?

Reply:
Indeed! However, the underlying JSON-RPC library can still raise `CatchableError` and there's no way to isolate this in code (likely because parts are macro-generated), As a result all RPC API modules has a top-level `{.push raises: [Defect, CatchableError].}` so we at least have visibility of this.